src/common/mfu_bz2_static.c

```diff
@@ -476,10 +476,10 @@ int mfu_decompress_bz2_static(const char* src_name, const char* dst_name)
     int64_t block_meta  = (int64_t)footer[0]; /* offset to start of block metadata */
     int64_t block_total = (int64_t)footer[1]; /* number of blocks */
     int64_t block_size  = (int64_t)footer[2]; /* max uncompressed size of a block */
-    int64_t data_size   = (int64_t)footer[3]; /* uncompressed size of all blocks */
+                                              /* unused - uncompressed size of all blocks */
```

Comment:
Let's keep the reference to the specific field and type for context in case we need it again later.
```suggestion
                                              /* (int64_t)footer[3] - unused - uncompressed size of all blocks */
```